### PR TITLE
Disabled save button in add category popup on guidelines page

### PR DESCRIPTION
**Pull request by shardulgusain**
#1717 

### Description
Implemented a function that will enable the save button only if user enter something on the form tab of add category window inside Guidelines and Resources option.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/js/app.js [hunk 1]
```diff
@@ -955,6 +955,16 @@ $(document).ready(function() {
 	  $(document).on("click", ".toggle-block-display", toggleBlockDisplay);
 	  $(document).on("change", ".send-mail-to-applicant", toggleApplicantMailEditor);
 });
+
+$(document).getElementById("name").onkeyup = ButtonEnable();
```
Comment (GauravGusain98) [inline resources/js/app.js:959]: The name of the id looks a bit generic. By looking at the name it's not getting clear, which element is this or what this element do? 

Please update the id name to a more specific name.

Comment (GauravGusain98) [reply resources/js/app.js:959]: Maybe `categoryName` would be a better name.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment.

File: resources/js/app.js (hunk 1 above)
Comment (shardulgusain) [reply resources/js/app.js]: Ok. Done.

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**